pkg/manifests/vllm/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
           - "8000"
           - "--enable-lora"
           - "--max-loras"
-          - "4"
+          - "2"
```

Comment:
Any reason for changing this?

Reply:
by mistake, I needed to change this to test on the L4s I have.